third_party/move/move-binary-format/src/file_format_common.rs

```diff
@@ -423,16 +424,22 @@ pub(crate) mod versioned_data {
     use std::io::{Cursor, Read};
     pub struct VersionedBinary<'a> {
         version: u32,
+        max_identifier_size: u64,
         binary: &'a [u8],
     }
 
     pub struct VersionedCursor<'a> {
         version: u32,
+        max_identifier_size: u64,
```

Comment:
There's something a bit smelly about having this as a field in a "VersionedCursor", but I suppose it's ok for now.  Next time we extend this we should think about a separate VersionedBinaryConfig struct instead.